### PR TITLE
Implement new versioning for Metronome

### PR DESCRIPTION
**Pull request by alenkacz**
We need unique name of every package built for Metronome. This will allow us to test them separately.

This is a proof that it works: https://s3.amazonaws.com/downloads.mesosphere.io/metronome/builds/0.5.72-8bb1367/metronome-0.5.72-8bb1367.tgz

It's the last build of this branch.

It's mostly the same stuff we did here for Marathon:

https://github.com/mesosphere/marathon/commit/e329bff2650c0c8b556d86136983ba97d10fa04d#diff-2af72f100c356273d46284f6fd1dfc08

JIRA issues: DCOS_OSS-2597

### CODE REVIEW

File: bin/install-protobuf.sh [hunk 1]
```diff
@@ -9,6 +9,9 @@ PLATFORM=`uname`
 echo $PLATFORM
 if [ "$PLATFORM" == 'Darwin' ]; then
    DOWNLOAD_URL="https://github.com/google/protobuf/releases/download/v$VERSION/protoc-$VERSION-osx-x86_64.zip"
+else
```
Comment (alenkacz) [inline bin/install-protobuf.sh]: I had to do this because I change the AMI for jenkins and that one does not have unzip

Comment (timcharper) [reply bin/install-protobuf.sh]: Or maybe we don't want to change this script more than necessary :) It was just kind of weird to begin with.

Comment (kensipe) [reply bin/install-protobuf.sh]: I like @timcharper recommendation

**Comment by kensipe** [comment]
rebased

File: version.sbt [hunk 2]
```diff
@@ -1 +0,0 @@
-version in ThisBuild := "0.5.3"
```
Comment (meln1k) [inline version.sbt]: Why not just delete this file then? :)

Comment (alenkacz) [reply version.sbt]: yep, I believe I did that...

**Pull request merged**